genai-perf/genai_perf/profile_data_parser/llm_profile_data_parser.py

```diff
@@ -282,8 +282,11 @@ def _get_tensorrtllm_engine_token_counts(
         for r in res_outputs:
             if isinstance(r["output_ids"], list):
                 token_ids += r["output_ids"]
-            else:
+            elif isinstance(r["output_ids"], int):
                 token_ids.append(r["output_ids"])
+            else:
+                # for the empty first/last responses
+                token_ids.append(0)
```

Comment:
@matthewkotila added a bug fix for empty first/last response (it will be and empty string and cause error).